python/lsst/pipe/base/pipelineIR.py

```diff
@@ -208,7 +214,7 @@ class ParametersIR:  # noqa: D405,D406,D407,D214 ("parameters" in code block)
     --------
     .. code-block:: yaml
 
-        parameters:
+        \u200bparameters:
```

Comment:
🤔 ZWSP?

Reply:
Yes, the commit message explains what's happening. pydocstyle is not clever enough to realize that `parameters:` is in a code block and is not a numpydoc directive. The ZWSP fools pydocstyle into thinking everything is fine...

Reply:
Ah, I reviewed the whole diff and did not look at commit messages.